### PR TITLE
[WNMGDS-250] Remove max and min from TextField prop documentation

### PR DESCRIPTION
**Pull request by bernardwang**
### Summary
We pass down any custom TextField props down to the input element, including `max`, `min`, `maxLength`, `minLength` etc. Since we don't do anything special to these attributes there's no reason to specifically call out `min` and `max` in our documentation. This PR removes them.

Note: we should consider adding extra documentation on the recommended way to do validation and user input requirements with our React form components

### Removed
`max` and `min` prop documentation

### CODE REVIEW

**Comment by line47** [comment]
Do we need to explain this in the documentation or no?  
> We pass down any custom TextField props down to the input element, including max, min, maxLength, minLength etc

**Comment by bernardwang** [comment]
@line47 The only reason why I didnt is because thats a default React behavior, and it also applies to all the other form components we have like `ChoiceList`. I feel it would be better to include guidance around validation, and mentioning how most of our form components accept native html validation attributes, but since they are abstractions above html elements it's not recommended to use them. 

Since that guidance isnt available yet, I'm still open to adding something small in the meantime! I'm just not sure where it would go...

**Comment by line47** [comment]
That makes sense and I agree about the validation guidance

File: packages/core/src/components/TextField/TextField.jsx [hunk 1]
```diff
@@ -234,17 +233,6 @@ TextField.propTypes = {
...
-   * `max` HTML input attribute
-   */
-  max: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /**
-   * `min` HTML input attribute
-   */
-  min: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-  /**
-   * Whether or not the text field is a multiline text field
-   */
```
Comment (dcloud) [inline packages/core/src/components/TextField/TextField.jsx]: Did you intend to remove the docstring for the multiline prop as well?

Comment (bernardwang) [reply packages/core/src/components/TextField/TextField.jsx]: oh oops! thanks for the catch

**Pull request merged**